cellbender/remove_background/tests/test_estimation.py

```diff
@@ -92,11 +93,12 @@ def test_mean_massive_m(log_prob_coo):
     new_shape = (coo.shape[0] + greater_than_max_int32, coo.shape[1])
     new_coo = sp.coo_matrix((coo.data, (new_row, coo.col)),
                             shape=new_shape)
+    print(new_coo)
```

Comment:
Either remove the print statement, or convert it to something more informative-- e.g. old matrix shape = ..., new matrix shape = ...